### ISSUE TITLE
Cool URIs & content negotiation

### ISSUE THREAD

**Issue by scuy** [codarchlab/ariadne-portal#23]
The portal should be able to resolve the URIs defined in the Gothenburg meeting.

It was decided that the following schema should be implemented:

```
http://registry.ariadne-infrastructure.eu/resource/{id}
http://registry.ariadne-infrastructure.eu/page/{id} (html)
http://registry.ariadne-infrastructure.eu/data/{id} (xml)
http://registry.ariadne-infrastructure.eu/resource/{id}.json

http://registry.ariadne-infrastructure.eu/provider/resource/{id} 
http://registry.ariadne-infrastructure.eu/provider/page/{id}
http://registry.ariadne-infrastructure.eu/provider/data/{id}
http://registry.ariadne-infrastructure.eu/provider/resource/{id}.json

http://registry.ariadne-infrastructure.eu/subject/resource/{id}
http://registry.ariadne-infrastructure.eu/subject/page/{id}
http://registry.ariadne-infrastructure.eu/subject/data/{id}
http://registry.ariadne-infrastructure.eu/subject/resource/{id}.json
```

See also: http://www.w3.org/TR/cooluris/

**Comment by scuy** [comment]
See if triple store is in place. Otherwise add dummy RDF.

**Comment by eafiontzi** [comment]
Triple store is in place but contains really old data based on an outdated schema.  I believe we should update it before we could use it.

**Comment by scuy** [comment]
Since we don't have data for providers and subjects in elasticsearch yet, I only implemented the URI handling for resources.

I would suggest replacing URIs of the form `http://registry.ariadne-infrastructure.eu/resource/{id}.json` with `http://registry.ariadne-infrastructure.eu/data/{id}.json` as I think that it makes more sense to be able to specify the format of the data and use the `/resource/{id}` endpoint only for the resource as a "thing".

However I implemented both endpoints in order to comply with the specification.
